src/main/java/sleepy/tasks/Event.java

```diff
@@ -0,0 +1,57 @@
+package sleepy.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * This class is a type of task.
+ *
+ * @author kjw142857
+ */
+public class Event extends Task {
```

Comment:
Would renaming to EventTask be better? Similar reason to the Deadline class comment